### DISCUSSION TITLE
The progress bar next to the song player's icon is ambiguous

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#25667] [Ideas]
I literally assumed that this bar tracks the progress of the song, and when I saw that when the song changed, the bar didn't, I thought it's a bug, but then noticed that it tracks the Master volume.

In my opinion it should probably track the progress of the song, and if not, then maybe it should be somehow clarified that it's tracking the Master volume

![image](https://github.com/ppy/osu/assets/82095491/5ea84283-e55b-4ebb-89d1-c2736c30e4a7)

**Comment by peppy** [comment]
Probably just needs a better icon. @arflyte
